internal/controllers/scheduling/controller.go

```diff
@@ -129,8 +129,10 @@ func (c *controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	if op == nil || inFlight >= c.concurrencyLimit {
 		return ctrl.Result{}, nil
 	}
-	if wait := time.Until(nextSlot); op.Reason.Deferred() && wait > 0 {
-		return ctrl.Result{RequeueAfter: wait}, nil
+	if !op.NotBefore.IsZero() {
```

Comment:
This is a bit criptic, what is it supposed to match on?

Reply:
lol yes this does require a minute to compute. Added a comment.